charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: 2.36.2
-version: 15.13.0
+version: 15.13.1
```

new feature should raise minor version.

```suggestion
version: 15.14.0
```